hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -62,6 +62,9 @@
         - 'gce-flannel':
             description: 'Run E2E tests on GCE using Flannel and the latest successful build. This suite is quarantined in a dedicated project because Flannel integration is experimental.'
             timeout: 90
+        - 'gce-examples':
```

Comment:
It's currently set to email $DEFAULT_RECIPIENTS, so this doesn't change anything.

I think I remember it used to email just you. Should I set you as the job owner and have it email you?